code/modules/reagents/chemistry/reagents/rechem.dm

```diff
@@ -0,0 +1,147 @@
+	var/slip_out_message = pick("silently fades in", "leaps out of thin air","appears", "walks out of an invisible doorway",\
+		"slides out of a fold in spacetime")
+	to_chat(user, "<span class='notice'>You try to align with the bluespace stream...</span>")
+	if(do_after(user, 20, target = src))
+		new /obj/effect/temp_visual/bluespace_fissure(get_turf(src))
+		new /obj/effect/temp_visual/bluespace_fissure(get_turf(linked))
+		user.forceMove(get_turf(linked))
+		user.visible_message("<span class='warning'>[user] [slip_in_message].</span>", null, null, null, user)
+		user.visible_message("<span class='warning'>[user] [slip_out_message].</span>", "<span class='notice'>...and find your way to the other side.</span>")
+
+/datum/reagent/zeolites
+	name = "Synthetic Zeolites"
+	description = "A liquid chemical with crystalline structure that can absorb radiation and convert it into healing."
+	taste_description = "sand"
+	color = "#4bc986" //RGB: 75, 201, 134
+	pH = 3.6
+
+	metabolization_rate = REAGENTS_METABOLISM * 3 //Fast boi
+
+/datum/reagent/zeolites/expose_obj(obj/exposed_obj, reac_volume)
+	var/datum/component/radioactive/contamination = exposed_obj.GetComponent(/datum/component/radioactive)
+	if(contamination && reac_volume >= 15)
+		qdel(contamination)
+		return
+	else if(contamination)
+		contamination.strength -= min(contamination.strength, round(reac_volume * pH * 8, 0.1)) //You actually can make it more efficient by using basic buffer on it
+
+/datum/reagent/zeolites/on_mob_life(mob/living/carbon/M)
+	. = ..()
+	var/datum/component/radioactive/contamination = M.GetComponent(/datum/component/radioactive)
+	if(M.radiation > 0) //Purges around
+		var/rad_to_remove = clamp(round((M.radiation / 150) * (15 * pH), 0.1), 0, M.radiation)
+		M.radiation -= rad_to_remove
+		var/power = 0.0075 * rad_to_remove
+		M.adjustOxyLoss(-3 * power, 0)
+		M.adjustBruteLoss(-power, 0)
+		M.adjustFireLoss(-power, 0)
+		M.adjustToxLoss(-power, 0, TRUE) //heals TOXINLOVERs
+		M.adjustCloneLoss(-power, 0)
+		for(var/i in M.all_wounds)
+			var/datum/wound/iter_wound = i
+			iter_wound.on_xadone(power)
+		REMOVE_TRAIT(M, TRAIT_DISFIGURED, TRAIT_GENERIC)
+
+	if(contamination && contamination.strength > 0)
+		contamination.strength -= min(contamination.strength, round(25 * pH, 0.1))
+
+/datum/reagent/nanite_b_gone
+	name = "Nanite bane"
+	description = "A stablised EMP that is highly volatile, shocking small nano machines that will kill them off at a rapid rate in a patient's system."
+	color = "#708f8f" //RBG: 112, 143, 143
+	overdose_threshold = 15
+	taste_description = "what can only be described as licking a battery."
+	pH = 9
+	can_synth = FALSE
+	metabolization_rate = REAGENTS_METABOLISM * 0.5
+
+/datum/reagent/nanite_b_gone/on_mob_life(mob/living/carbon/C)
+	var/datum/component/nanites/N = C.GetComponent(/datum/component/nanites)
+	if(isnull(N))
+		return ..()
+	N.adjust_nanites(-volume) //0.5 seems to be the default to me, so it'll neuter them.
```

Comment:
looking at the comment, did you mean to use the nanite default regeneration volume here? As it's written right now, it will remove an amount equal to the reagent volume every tick, which is very strong

Reply:
Making this stuff requires T2 reagents and if the chemists fails to brew it propperly, it just EMPs/shocks them. Mass producing those will require constant supply of synthflesh and temperature control